src/data/logoIcons.json

```
@@ -15,9 +15,9 @@
   },
   "icons": [
     {
-      "name": "logo-7H",
+      "name": "logo-RA",
```

RA is the wrong IATA - needs to stay as 7H.
```suggestion
      "name": "logo-7H",
```